sources/packages/forms/src/form-definitions/reportscholasticstandingchange.json

```diff
@@ -3344,121 +3400,110 @@
                   "components": [
                     {
                       "label": "Date of change",
-                      "format": "yyyy-MM-dd",
+                      "labelPosition": "top",
+                      "labelWidth": "",
+                      "labelMargin": "",
                       "placeholder": "Select date",
-                      "tableView": false,
-                      "enableMinDateInput": false,
-                      "datePicker": {
-                        "disableWeekends": false,
-                        "disableWeekdays": false,
-                        "showWeeks": true,
-                        "startingDay": 0,
-                        "initDate": "",
-                        "minMode": "day",
-                        "maxMode": "year",
-                        "yearRows": 4,
-                        "yearColumns": 5,
-                        "minDate": null,
-                        "maxDate": null
-                      },
-                      "enableMaxDateInput": false,
-                      "enableTime": false,
-                      "customOptions": {
-                        "dateFormat": "yyyy-MM-dd"
-                      },
-                      "validate": {
-                        "required": true,
-                        "custom": "",
-                        "customPrivate": false,
-                        "strictDateValidation": false,
-                        "multiple": false,
-                        "unique": false
-                      },
-                      "key": "dateOfChange",
-                      "type": "datetime",
-                      "input": true,
+                      "description": "",
+                      "tooltip": "",
+                      "prefix": "",
+                      "suffix": "",
                       "widget": {
                         "type": "calendar",
-                        "displayInTimezone": "viewer",
-                        "locale": "en",
-                        "useLocaleSettings": false,
                         "allowInput": true,
-                        "mode": "single",
+                        "clickOpens": true,
+                        "enableDate": true,
                         "enableTime": false,
+                        "mode": "single",
                         "noCalendar": false,
                         "format": "yyyy-MM-dd",
+                        "dateFormat": "yyyy-MM-dd",
+                        "useLocaleSettings": false,
                         "hourIncrement": 1,
-                        "minuteIncrement": 1,
+                        "minuteIncrement": 5,
```

Comment:
mm... its date only. do we need `minuteIncrement`

Reply:
These are the default properties generated by the widget and because we set enableTime to false, these properties are ineffective.